### PR TITLE
Mend vulnerability scan pipeline

### PR DESCRIPTION
**Pull request by Moran-k**
Scan all images in `komodor-agent` values.yaml file using mend.

The scan script:
* scan for critical vulnerabilities only
* Skip critical vulnerabilities without a fixed version
* Ignore vulnerabilities that are in the `exclude.yaml`

In case a critical vulnerability is found, the build will fail.
Build failure notification will be sent to `#critical-security-vulns` slack channel

### CODE REVIEW

**Comment by Moran-k** [comment]
CU-860rpy6rn

File: .buildkite/vulnerability-scan/tests.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+EMPTY_EXCLUDE_CVES = []
+
+EXCLUDE_CVES = [
+    {
+        'SystemPath': '/usr/bin/pluto',
+        'CVEs': ['CVE-2023-24540'],
+        'MuteUntil': datetime.date(2050, 1, 1)
+    },
+]
```
Comment (MickaelAlliel) [inline .buildkite/vulnerability-scan/tests.py:23]: Is this not easier to have in the exclude.yaml which is currently empty?

**Review by MickaelAlliel — APPROVED** [review]
The slack notification is done via Mend? As i dont see anything in this script

**Comment by Moran-k** [comment]
> The slack notification is done via Mend? As i dont see anything in this script

No, For start, i will just add the buildkite notification on failed build

**Pull request merged**